### PR TITLE
upgrade the dependencies and crush lint erros

### PR DESCRIPTION
**Pull request by jks8787**
upgrade dependencies to latest available version (based on este)

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -26,25 +28,6 @@ const runEslint = () => {
   .pipe(eslint.format());
 };
 
-function buildReact(minify) {
```
Comment (jaslakson) [inline gulpfile.babel.js:29]: I don't think we want to get rid of these tasks.

**Comment by jaslakson** [comment]
Other than the gulp tasks that were removed, this is :rocket: :+1:

**Comment by LaureMFischer** [comment]
:+1:

File: gulpfile.babel.js (hunk 1 above)
Comment (jks8787) [inline gulpfile.babel.js:29]: :rabbit:  good catch

**Comment by jaslakson** [comment]
:+1:

**Pull request merged**